polaris-patterns/src/components/DatePickerTextField/DatePickerTextField.tsx

```diff
@@ -48,7 +50,17 @@ const DISPLAY_DATE_FORMAT: Intl.DateTimeFormatOptions = {
   timeZone: 'UTC',
 };
 
-export function DatePickerTextField({
+export function DatePickerTextField(props: Props) {
+  const manager = useShopifyI18nManager();
+
+  return (
+    <I18nContext.Provider value={manager}>
+      <DatePickerTextFieldComponent {...props} />
+    </I18nContext.Provider>
+  );
+}
+
```

Comment:
Same as above, wrapping this component in its own `I18nContext.Provider` and passing in the locale, country, etc. from my own context. This enables these components to be used in isolation and the consuming app doesn't have to use `@shopify/react-i18n`